pkg/upgrade/upgrades/tenant_table_migration_test.go

```diff
@@ -60,10 +61,21 @@ func TestUpdateTenantsTable(t *testing.T) {
 	var (
```

Comment:
should this test maybe use the tenants table after the upgrade? Perhaps create a tenant and then query the table and validate the structure?